service/controller/chart/v1/resource/tiller/resource.go

```diff
@@ -56,7 +57,12 @@ func (r *Resource) ensureTillerInstalled(ctx context.Context) error {
 		"spec.template.spec.tolerations[0].operator=Exists",
 	}
 	err := r.helmClient.EnsureTillerInstalledWithValues(ctx, values)
-	if err != nil {
+	if helmclient.IsTooManyResults(err) {
+		r.logger.LogCtx(ctx, "level", "debug", "message", "currently too many tiller pods")
```

Comment:
Does it make sense to be more precise here?

```suggestion
		r.logger.LogCtx(ctx, "level", "debug", "message", "currently too many tiller pods due to upgrade")
```

Reply:
Good shout. Done.